demo/rube_testbed/loadrube.js

```diff
@@ -37,7 +37,11 @@ function loadBodyFromRUBE(bodyJso, world) {
     else
         bd.set_position(new Box2D.b2Vec2(0,0));
 
-    bd.set_gravityScale(bodyJso.gravityScale || 1);
+    if (bodyJso['gravityScale'] != undefined) {
```

Comment:
There would be no semantic difference if you wrote `if (bodyJso.gravityScale`, though? As appears on the next line. (The only difference that could happen is if closure compiler ran on the code, then it treats strings differently.)

Also, you probably want `===` and not `==` here (as e.g. `null != undefined = false`).